src/routes/graphql/schema.ts

```diff
@@ -0,0 +1,616 @@
+import {
```

Comment:
You did a great job! However, the codebase would really benefit from being split into smaller modules/files as some of the created files are huge which decreases readability